config/overlays/odh/params.env

```
@@ -1,7 +1,7 @@
 IMAGES_MODELREGISTRY_OPERATOR=quay.io/opendatahub/model-registry-operator:latest
 IMAGES_GRPC_SERVICE=quay.io/opendatahub/mlmd-grpc-server:latest
 IMAGES_REST_SERVICE=quay.io/opendatahub/model-registry:latest
-IMAGES_OAUTH_PROXY=quay.io/openshift/origin-oauth-proxy:latest
+IMAGES_OAUTH_PROXY=registry.redhat.io/openshift4/ose-oauth-proxy@sha256:6af6327b69bc892188bd09a343758dd2d397f73591084a4876543dae816318d8
```

@dhirajsb I wonder if this manifest list digest ref should be used instead:

```suggestion
IMAGES_OAUTH_PROXY=registry.redhat.io/openshift4/ose-oauth-proxy-rhel9@sha256:3bb1803cae39a31cc87bcf8affee0ad2576f1d2774a6c3de556adbbbc2687abc
```

The one that you reference is from v4.12, which doesn't seem to have images for any architecture than amd64. The one I propose is the latest from v4.18 and contains images for all of the arches. While that might not be a problem that you have right now, I imagine it will be soon :slightly_smiling_face:

---

Thanks @grdryn , merged.

---

we wanna use the one from rhel9?